analyzers/tests/SonarAnalyzer.Test/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -403,22 +403,31 @@ public void Method()
 
     [TestMethod]
     public void Analyze_Severity_ExecutesWhenMore() =>
-        Verify(@"string s = null;   // Noncompliant    {{Message for SAll}}
-                                        // Noncompliant@-1 {{Message for SMain}}
-                    s.ToString();       // Compliant, should not raise S2259",
+        Verify("""
+            string s = null;    // Noncompliant    {{Message for SAll}}
+                                // Noncompliant@-1 {{Message for SMain}}
+            s.ToString();       // Compliant, should not raise S2259
+            """,
+            expectingIssues: true,
```

Comment:
```suggestion
```
It's default

Reply:
I cannot omit because of the params that follows.